### ISSUE TITLE
Move to D-guild's github org

### ISSUE THREAD

**Issue by fnurkla** [GitThisJoel/intakt#6]

**Comment by GitThisJoel** [comment]
I totally agree. I would say that once we get both Swish and Zettle to work properly we can start moving this repo to D-sek.

**Comment by GitThisJoel** [comment]
@fnurkla have set this into motion again, will start to look into it again
